### PR TITLE
Update to difference/change significance formulas

### PR DESCRIPTION
**Pull request by EricaMaurer**
Update to difference and change significance formulas to calculate a CV for each estimate/MOE pair to determine reliability.

I think I updated each instance, but please check that I didn't miss any other significance calculations in the code.

### CODE REVIEW

File: query-helpers/profile.js [hunk 1]
```diff
@@ -142,20 +142,16 @@ const buildSQL = function buildSQL(profile, ids, compare) {
       *,
       -- significant --
       CASE
-        WHEN ABS(SQRT(POWER(coalesce(m, 0) / 1.645, 2) + POWER(coalesce(comparison_m, 0) / 1.645, 2)) * 1.645) > ABS(comparison_sum - sum) THEN false
-        ELSE true
+        WHEN ((((difference_m) / 1.645) / ABS(difference_sum)) * 100) < 20
```
Comment (allthesignals) [inline query-helpers/profile.js]: `difference_m` is defined in the same selection so we'll need to subquery

**Comment by allthesignals** [comment]
There are a bunch of errors with division by zero and subquerying that i'm debugging now, but will take a little time .

**Comment by EricaMaurer** [comment]
@allthesignals thank you. Let me know if you need me to look at anything

**Comment by allthesignals** [comment]
Tests pass, I'm going to try and deploy @EricaMaurer

**Pull request merged**